manifests/0000_51_olm_00-olm-operator.yml

```diff
@@ -7,8 +7,8 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: "OperatorLifecycleManagerV1"
```

Comment:
@perdasilva Because v0's capability name is OperatorLifecycleManager, I do not suggest to name v1's capability name as OperatorLifecycleManagerV1.  
If some existing code check capability's name with `contains` or `regrexp`, OperatorLifecycleManagerV1 could be treated as v0 possible, so suggest not to name OperatorLifecycleManagerV1 to avoid this potential problem. (if some existing code is strong, no such issue. but I still suggest not to name it to avoid potential issue.).

by the way, if v0's name is OperatorLifecycleManagerV0, it is ok to name v1 as OperatorLifecycleManagerV1.

Thanks

Reply:
I'm really sorry I missed this, because it raises a valid point. But, I'm afraid now it's too late. We've put through the changes to openshift/api and the capability annotation already merged in a different PR =SS

Reply:
never mind, and let's move on

Reply:
Mitigating factor: all official code for capability name is based on exact string matching. So this shouldn't be a problem for OCP proper. I _might_ be a problem if clients are doing things that way, but the enhancement for the capability feature doesn't mention substring matching being a valid way to compose capabilities, so I think we are in the clear.

Reply:
ok, thanks